applications/data_analysis/humid_heat_metrics.ipynb

```diff
@@ -0,0 +1,1770 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.    ax.annotate(
</pre><p>Also, is labeling Chicago necessarily adding value to this notebook? We haven't mentioned it elsewhere in the notebook that I'm seeing.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/geocat-applications/pull/66/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
At this point, not really - but I chose this date because of being the peak of a famous heatwave in Chicago. I think I mention this heatwave in my intro, and it's nice to see that Chicago is one of the hotter regions.

I was going to try and recreate the Li analysis for that heatwave (they looked at 9 or so heatwaves, but I was going to only do one). Until then I can remove it.